src/monitor.rs

```diff
@@ -113,6 +113,14 @@ pub trait MonitorHandleProvider: AsAny + fmt::Debug + Send + Sync {
     #[cfg_attr(not(web_platform), doc = "detailed monitor permissions.")]
     fn position(&self) -> Option<PhysicalPosition<i32>>;
 
+    /// Returns physical size of monitor in millimeters.
+    ///
```

Comment:
Nit: Specify what `None` means.

```suggestion
    ///
    /// Returns `None` if the size of the monitor could not be determined.
    ///
```